### PR TITLE
[WFLY-11759] Exclude problematic Undertow interfaces from ServiceLoader calls when testing management model transformation

### PR DESCRIPTION
**Pull request by bstansberry**
This also reverts https://github.com/wildfly/wildfly/pull/12098 which was a workaround pending this solution.

https://issues.jboss.org/browse/WFLY-11759

### CODE REVIEW

**Comment by bstansberry** [comment]
Tests of this in combination with the proposed Undertow upgrade in core (https://github.com/wildfly/wildfly-core/pull/3820):

https://ci.wildfly.org/viewType.html?buildTypeId=WF_WildFlyCoreIntegrationExperiments&branch_WF=%2FWFLY-11759%3B%2Fbstansberry%3A%2FWFCORE-4526%3B%2Ffl4via

https://ci.wildfly.org/viewType.html?buildTypeId=WF_WildFlyCoreIntegrationExperimentsWindows&branch_WF=%2FWFLY-11759%3B%2Fbstansberry%3A%2FWFCORE-4526%3B%2Ffl4via

File: undertow/src/test/resources/org/wildfly/extension/undertow/undertow-7.1-transformers.xml [hunk 1]
```diff
@@ -36,19 +36,17 @@
...
             <filter-ref name="headers" priority="${some.priority:10}"/>
             <filter-ref name="404-handler"/>
-            <!--
             <filter-ref name="static-gzip" predicate="path-suffix('.js')"/>
-            -->
          </location>
-         <!--<access-log directory="${jboss.server.server.dir}" pattern="REQ %{i,test-header}" predicate="not path-suffix(*.css)" prefix="access" rotate="false"/>-->
-         <access-log directory="${jboss.server.server.dir}" pattern="REQ %{i,test-header}" prefix="access" rotate="false"/>
+         <access-log directory="${jboss.server.server.dir}" pattern="REQ %{i,test-header}" predicate="not path-suffix(*.css)" prefix="access" rotate="false"/>
+         <!-- <access-log directory="${jboss.server.server.dir}" pattern="REQ %{i,test-header}" prefix="access" rotate="false"/> -->
```
Comment (kabir) [inline undertow/src/test/resources/org/wildfly/extension/undertow/undertow-7.1-transformers.xml]: Can't this one simply be removed? The other line seems to contain everything and more

**Review by kabir — APPROVED** [review]
Apart from my nitpick it looks good

File: undertow/src/test/resources/org/wildfly/extension/undertow/undertow-7.1-transformers.xml (hunk 1 above)
Comment (bstansberry) [reply undertow/src/test/resources/org/wildfly/extension/undertow/undertow-7.1-transformers.xml]: This commit isn't mine, it's from #12363 so I don't want to change it.

Comment (fl4via) [reply undertow/src/test/resources/org/wildfly/extension/undertow/undertow-7.1-transformers.xml]: @kabir good catch, I'll submit a new PR to remote those, sorry for the mistake

Comment (fl4via) [reply undertow/src/test/resources/org/wildfly/extension/undertow/undertow-7.1-transformers.xml]: I mean, to remove those.

**Pull request merged**